tests/test_populate.py

```diff
@@ -329,6 +328,54 @@ def test_build_electrode_layouts(pipeline):
         ), f"probe type '{probe_type}' electrode layout does not match"
 
 
+def test_quality_metrics_populate(pipeline):
+    """
+    Populate ephys.QualityMetrics and compare values with metrics.csv
```

Comment:
```suggestion
    Populate the ephys.QualityMetrics table and compare values with the `metrics.csv` file.
    Run the `demo_prepare.ipynb` notebook, prior to running this test.
```

Reply:
Thanks, looks like some of the changes we made yesterday got lost since I removed the codespace without committing them.